app/src/main/kotlin/com/wire/android/migration/MigrationManager.kt

```diff
@@ -373,8 +381,20 @@ fun MigrationData.Result.Failure.toData(): Data = when (this) {
     MigrationData.Result.Failure.NoNetwork -> workDataOf(
         MigrationData.Result.Failure.KEY_FAILURE_TYPE to MigrationData.Result.Failure.FAILURE_TYPE_NO_NETWORK
     )
+
+    is MigrationData.Result.Failure.Unknown -> {
+        if (this.throwable != null) {
+            workDataOf(
```

Comment:
`MigrationData.Result.Failure.KEY_FAILURE_TYPE` is not passed for this type of failure